book/website/collaboration/stakeholder-engagement.md

```diff
@@ -0,0 +1,116 @@
+Strategy alignment goes deeper than agreeing on day to day work - it asks what the team chooses to prioritise, what they value, what motivates them, and ultimately what real-world impact they hope to see from their work.
+A helpful model to inform this exercise is Simon Sinek's [Golden Circle method](https://www.smartinsights.com/digital-marketing-strategy/online-value-proposition/start-with-why-creating-a-value-proposition-with-the-golden-circle-model/), where the team approaches their work in the following order:
+* Why: Why are we doing what we're doing? 
+    * This is the ultimate 'North Star' or 'vision' of the project - the fundamental, emotive reason the work is being done
+* How: How do we achieve our mission? 
+    * This question concerns the broad focus areas, or 'pillars' that setup up the project to achieve the vision
+* What: What do we actually do?
+    * This question takes the why and the how, and identifies key activities that teams can undertake to achieve their goals
+
+It is important to take these conversations in order, so that strategy alignment can build from the bottom-up.
+
+### Example resources
+- [Further RAM Guidance](https://github.com/alan-turing-institute/research-application-management/blob/main/docs/team_strategy.md) on facilitating Why/How/What conversations
+
+### RAM insights
+* Using this basic infrastructure for the conversation helps provide focus and clarity and can lead to a better outcome
+* By not overstructuring the conversation, you will also allow people the space to express their thoughts freely. We feel this is an important approach as it allows open conversations to happen and things to emerge that otherwise might not
+* Strong facilitation skills are crucial, including reading the room and identifying patterns in the conversation. For more on  effective facilitation, check out:
+    * {ref}`Chairing Events <cl-chairing>`
+    * {ref}`Tools for Collaboration <cl-event-tools>`
+
+(cl-stakeholder-user-design)=
+## User-centric design
+### Key question: How can research outputs benefit from a user-centric design process?
+
+Although many research tools are created for a very narrow purpose, we encourage researchers to consider the user experience (UX) throughout the process and to incorporate UX design principles to make the tool more accesible and usable for all potential users. 
+Activities such as user journey mapping and user testing can greatly enhance your understanding of who is using the research tool and how they employ the tool in a real-world scenario.
+By considering potential users of the tool from the early stages of a project, you can include features that will make the tool more accessible to potential stakeholders and therefore more likely to be used and applied in practice. 
+
+Existing resources can also benefit greatly from making improvements to the UX by establishing heuristics to define criteria for usability and to identify areas in need of improvement.
```

Comment:
Link out to a definition of heuristics? V technical term

Reply:
Is heuristics a controversial term? Not sure but just to check!